src/suites/cts/validation.spec.ts

```diff
@@ -0,0 +1,673 @@
+export const description = `
+Validation tests.
```

Comment:
There are going to be a LOT of validation tests. I suggest splitting them per object tested, or even better per function tests. (in this case mostly createBindGroupLayout and createBindGroupLayout).

Reply:
Yes, definitely need to split up this file. I'm not sure what directory structure is best: validation/.../createBindGroup or .../createBindGroup or .../createBindGroup/validation. But I'll deal with that later; pick whatever you want.